src/wallet/spend.cpp

```diff
@@ -432,10 +430,10 @@ std::optional<SelectionResult> SelectCoins(const CWallet& wallet, const std::vec
     {
         for (const COutput& out : vCoins) {
             if (!out.m_spendable) continue;
-            /* Set depth, from_me, ancestors, and descendants to 0 or false as these don't matter for preset inputs as no actual selection is being done.
+            /* Set ancestors and descendants to 0 as these don't matter for preset inputs as no actual selection is being done.
              * positive_only is set to false because we want to include all preset inputs, even if they are dust.
              */
-            preset_inputs.Insert(out.GetInputCoin(), 0, false, 0, 0, false);
+            preset_inputs.Insert(out, 0, 0, false);
```

Comment:
nit: document the remaining bools while touching this

Reply:
Done